GAME/script.js

```diff
@@ -0,0 +1,180 @@
+$(function () {
+
+  //saving dom objects to variables
+  var container = $('#container');
+  var bird = $('#bird');
+  var pole = $('.pole');
+  var pole_1 = $('#pole_1');
+  var pole_2 = $('#pole_2');
+  var pole_3 = $('#pole_3');
+  var pole_4 = $('#pole_4');
+  var score = $('#score');
+  var speed_span = $('#speed');
+  var restart_btn = $('#restart_btn');
+
+  //saving the initial setup
+  var container_width = parseInt(container.width());
+  var container_height = parseInt(container.height());
+  var pole_initial_position = parseInt(pole.css('right'));
+  var pole_initial_height = parseInt(pole.css('height'));
+  var bird_left = parseInt(bird.css('left'));
+  var bird_height = parseInt(bird.height());
+  var speed = 10;
+
+  //other declarations
+  var go_up = false;
+  var score_updated = false;
+  var game_over = false;
+
+  //set interval function which runs every 40 milliseconds
+  var the_game = setInterval(function () {
```

Comment:
If you'd like to freeze the bird at the beginning, you need to add some flag here (flag changes when the player press Space).

Reply:
Hi Alina, sorry i just saw this now, I just let it click to start instead. 
Just pushed my updated version through now but I will have a look at what you said with you tomorrow if possible. 
Thank you.